Plugins/KawaiiPhysics/Source/KawaiiPhysicsEd/Public/KawaiiPhysicsEditMode.h

```diff
@@ -19,6 +19,11 @@ class FKawaiiPhysicsEditMode : public FKawaiiPhysicsEditModeBase
 public:
 
 	FKawaiiPhysicsEditMode();
+
+	virtual void Enter() override {}
+	virtual void Exit() override {}
+	virtual void AddReferencedObjects(FReferenceCollector& Collector) override {}
+	
```

Comment:
Here also needs a `virtual bool SupportsPoseWatch() override { return false; };` for it to complie.